experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl

```diff
@@ -122,6 +122,82 @@ function defining_section_parametrization(m::AbstractFTheoryModel)
 end
 
 
+@doc raw"""
+    classes_of_model_sections(m::AbstractFTheoryModel)
+
+Return the divisor classes of all model sections.
+
+```jldoctest
+julia> B3 = projective_space(NormalToricVariety, 3)
+Normal toric variety
+
+julia> w = torusinvariant_prime_divisors(B3)[1]
+Torus-invariant, prime divisor on a normal toric variety
+
+julia> t = literature_model(arxiv_id = "1109.3454", equation = "3.1", base_space = B3, defining_classes = Dict("w" => w), completeness_check = false)
+Construction over concrete base may lead to singularity enhancement. Consider computing singular_loci. However, this may take time!
+
+Global Tate model over a concrete base -- SU(5)xU(1) restricted Tate model based on arXiv paper 1109.3454 Eq. (3.1)
+
+julia> classes_of_model_sections(t)
+Dict{String, ToricDivisorClass} with 9 entries:
+  "a21" => Divisor class on a normal toric variety
+  "a6"  => Divisor class on a normal toric variety
+  "a3"  => Divisor class on a normal toric variety
+  "w"   => Divisor class on a normal toric variety
+  "a2"  => Divisor class on a normal toric variety
+  "a1"  => Divisor class on a normal toric variety
+  "a43" => Divisor class on a normal toric variety
+  "a4"  => Divisor class on a normal toric variety
+  "a32" => Divisor class on a normal toric variety
+```
+"""
+@attr Dict{String, ToricDivisorClass} function classes_of_model_sections(m::AbstractFTheoryModel)
+  @req hasfield(typeof(m), :explicit_model_sections) "explicit_model_sections not supported for this F-theory model"
```

Comment:
Two questions here:
1. Should the two strings in these `@req`s start with capital letters to match the other error messages in OSCAR?
2. Do we actually need to check if `explicit_model_sections` exists in order to return classes of model sections? Certainly, taking the classes of the explicit sections is one way to determine the classes, but we could not have made the explicit sections in the first place if we didn't already know the classes. The classes are already stored as a grading matrix for all the literature models, and for model families we may not have explicit sections but we still know their classes. It seems the classes should have their own attribute, and that should be checked here.